packages/yew/src/virtual_dom/vnode.rs

```diff
@@ -35,6 +35,16 @@ impl VNode {
         }
     }
 
+    /// Returns, if the VNode has a key without needlessly cloning the key
```

Comment:
```suggestion
    /// Works out whether the `VNode` has a key (without cloning the key).
```

Reply:
Or maybe:
```suggestion
    /// Returns true if the `VNode` has a key without needlessly cloning the key.
```

Reply:
Yeah either would work.

Reply:
I'd do ```[`VNode`]``` instead so links to the type.

Reply:
This was done before I discovered doc links. I'll do that for the entire PR then.